lntest/itest/lnd_payment_test.go

```diff
@@ -374,19 +375,36 @@ func testAsyncPayments(net *lntest.NetworkHarness, t *harnessTest) {
 		}
 
 		return true
-	}, time.Second*5)
+	}, defaultTimeout)
 	if err != nil {
 		t.Fatalf("failed to assert alice's pending htlcs and/or remote/local balance")
 	}
 
 	// Wait for Bob to receive revocation from Alice.
-	time.Sleep(2 * time.Second)
+	var bobChan *lnrpc.Channel
```

Comment:
Great to get rid of a static sleep!

Reply:
Looks like Travis gotten slower these days as this flake I've never seen before.